examples/policy_checker/license_policy_check.bzl

```diff
@@ -69,7 +69,6 @@ _license_policy_check = rule(
             doc = """Target to collect LicenseInfo for.""",
```

Comment:
Should this be a label_list that we iterate over?

Reply:
Yes, that would allow more flexibility for users applying the same license check to multiple targets.

Reply:
@aiuto please take another look